lib/gui/os/dialog/services/dialog.js

```diff
@@ -113,20 +113,28 @@ module.exports = function($q, SupportedFormatsModel) {
    * });
    */
   this.showWarning = (options) => {
+    const BUTTONS = [
+      options.confirmationLabel,
```

Comment:
Should we be setting defaults for `.confirmationLabel` and `.rejectionLabel` in case they don't exist?